datadog_checks_dev/datadog_checks/dev/tooling/utils.py

```diff
@@ -169,6 +169,15 @@ def get_metadata_file(check_name):
     return os.path.join(get_root(), check_name, 'metadata.csv')
 
 
+def get_saved_views(check_name):
+    paths = load_manifest(check_name).get('assets', {}).get('saved_views', {})
+    views = []
+    for path in paths.values():
+        view = os.path.join(get_root(), check_name, path)
```

Comment:
For Windows you'll need something like https://github.com/DataDog/integrations-core/blob/be94cde0262ad87c33597e5d746fa817c035cc2a/datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/dashboards.py#L33

Reply:
Updated! Let me know if it fixed it for you